plugins/postcss-stepped-value-functions/test/basic.expect.css

```diff
@@ -41,6 +41,12 @@
 	right: 8px;
 }
 
+.unit {
+	top: 37.79527559055118px;
```

Comment:
It looks weird to round something and to get a decimal, but this is rounded to `1cm`.

<img width="278" alt="Screenshot 2023-02-14 at 19 17 00" src="https://user-images.githubusercontent.com/11521496/218823265-b3f2d1ef-d185-4782-bc05-880cdf3c84fe.png">